### DISCUSSION TITLE
How can I use a vimscript plugin

### DISCUSSION THREAD

**Discussion by ValBlaze** [LazyVim/LazyVim#1236] [Q&A] [answered]
I have this plugin I want to use written in vimscript: https://github.com/elkowar/yuck.vim How can I use this with lazy.nvim?

**Comment by amaanq** [comment, marked as answer]
just add it to your spec in any plugin file, though nvim-treesitter does have yuck support

**Reply by ValBlaze** [reply]
sorry, what is my spec?

**Reply by ValBlaze** [reply]
nevermind I figured it out, thank you!

**Comment by tonmoy998** [comment]
just put the name for example in vim plug : Plugin 'christoomey/vim-tmux-navigator'
put it on your lua file only with {'christoomey/vim-tmux-navigator'} return
